src/Main.java

```diff
@@ -0,0 +1,49 @@
+import zoo.*;
```

Comment:
- [ ] לקרוא על סוגים של רשימות
- [ ] לקרוא על exceptions
- [ ] לקרוא על איטרטורים וforeach
- [ ] לקרוא על פולימורפיזם
- [ ] לקרוא על asList והבעיות שהוא יכול ליצור
- [ ] למה אי אפשר לרשת מיותר ממחלקה אחת

Reply:
- [ ] Throws במיין

Reply:
- [ ] logger

Reply:
- [ ] bubbling exceptions

Reply:
- [ ] Split the packages

Reply:
> * [x]  Throws במיין

קראתי על זה ובגדול אני מבין שזה לא כל כך טוב. במקום שאני אטפל בשגיאה אני נותן למשהו מחוץ לשליטתי לטפל בה